commands/audit/sca/python/python_test.go

```diff
@@ -16,11 +19,11 @@ func TestBuildPipDependencyListSetuppy(t *testing.T) {
 	_, cleanUp := sca.CreateTestWorkspace(t, filepath.Join("projects", "package-managers", "python", "pip", "pip", "setuppyproject"))
 	defer cleanUp()
 	// Run getModulesDependencyTrees
-	rootNode, uniqueDeps, err := BuildDependencyTree(&AuditPython{Tool: pythonutils.Pip})
+	rootNode, uniqueDeps, err := BuildDependencyTree(nil, coreutils.Pip, &xrayutils.AuditBasicParams{})
```

Comment:
no need for change, should pass `AuditPython`

Reply:
well it need to get params in order to save download urls for later use in curation, If I pass params and audit python it will duplicate some fields, though its possible, I'm not sure its good practice.

Reply:
Fixed